azerothcore/azerothcore-wotlk
# Bugs/445

## Changes Proposed:
- Update SmartAI causing 4690 to Move Randomnly instead of standing still 

## Issues Addressed:
- Closes 
https://github.com/chromiecraft/chromiecraft/issues/445
https://github.com/azerothcore/azerothcore-wotlk/issues/5366

## SOURCE:
None- Fixed based on the Bug report and that I dont remember any mob just standing still
## Tests Performed:
- Visited 4688 4689 and 4690 they all stagger around like drunken salors now. 

## How to Test the Changes:
1. Apply SQL script
2. .go xyz 1789.01	92.1613	2.94131
3. Ensure 4688 is moving randomly
4. .go xyz 1104.43	100.077	5.36425
5. Ensure 4689 is moving Randomly
6. .go xyz 1799.25	66.8346	3.06855
7. ensure 4690 is moving randomly

## Known Issues and TODO List:


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
